main.go

```diff
@@ -57,6 +63,8 @@ func main() {
 	flag.StringVar(&leaderElectionID, "leader-election-id", "signer-venafi-leader-election",
 		"The name of the configmap used to coordinate leader election between controller-managers.")
 	flag.BoolVar(&debugLogging, "debug-logging", true, "Enable debug logging.")
+	flag.StringVar(&signerName, "signer-name", "example.com/foo", "Only sign CSR with this .spec.signerName.")
```

Comment:
This command line argument is difficult to override in kustomize without copying all the parent defined command line arguments.
Maybe that would be made easier if we read signerName from a config file instead.